src/service/comparison_vs_peers/comparison_vs_peers_service.py

```diff
@@ -105,6 +130,37 @@ def get_comparison_vs_data(self, data: dict, access: bool) -> list:
 
         return rule_of_40
 
+    def filter_by_conditions(self, data: dict, **conditions) -> dict:
+        companies_filtered = dict()
+        if (
+            conditions.__contains__("size_cohort") is True
+            and conditions.__contains__("margin_group") is True
```

Comment:
The python documentation recommends avoiding comparing booleans to True or False as indicated in the example:

- Recommended ->  if greeting:
- No -> if greeting == True:
- Worse -> if greeting is True: 

In this case you can try to compare the `conditions.__contains__("xxxx")` like this: 
`
if (conditions.__contains__("size_cohort")
            and conditions.__contains__("margin_group")):
`
In case you need to compare a False value you can do it like this:
`
if ( not conditions.__contains__("size_cohort")
            and conditions.__contains__("margin_group")):
`


You can find more info [here](https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html):